chart/values.yaml

```diff
@@ -3011,6 +3011,7 @@ config:
   celery:
     flower_url_prefix: '{{ ternary "" .Values.ingress.flower.path (eq .Values.ingress.flower.path "/") }}'
     worker_concurrency: 16
+    sync_parallelism: '{{ include "cpu_count" (((.Values.scheduler).resources).limits).cpu }}'
```

Comment:
Thanks for adding this automation for the option.

Are you sure that `sync_paralellism` is the right config key you are looking for? This PR descriptions rather sounds like you are referring to `worker_concurrency`. I would expect this config key to be set.